src/mlpack/methods/adaboost/adaboost_impl.hpp

```diff
@@ -242,7 +244,8 @@ void AdaBoost<WeakLearnerType, MatType>::Classify(
     const MatType& test,
     arma::Row<size_t>& predictedLabels,
     arma::mat& probabilities)
-{
+{ 
+  
```

Comment:
Please keep changes to existing code base minimal. Only change if you *absolutely* need to, this saves reviewer's time since Github reports every little change. I don't think this \n existed before, so remove here.